doc/feature/httpServer.md

```diff
@@ -47,12 +47,20 @@ tracks the implementation.
         virtual void fromJSON( const std::string& json );
         virtual std::string toJSON() const;
 
-        /** Called after the object has been updated. */
-        virtual void notifyUpdated() {}
+        /**
+         * Set a new function called after the object has been updated.
+         *
+         * @return the previously set function.
+         */
+        ChangeFunc setUpdatedFunction( const ChangeFunc& func );
```

Comment:
This is already merged, this is just the documentation update. If you want it changed, submit a PR. ;)